tasks/sanity_test.py

```diff
@@ -0,0 +1,62 @@
+import multiprocessing
+import pytest
+
+def sanity_test(algorithm, test_imgs):
+    """ 
+    Runs a sanity test on the algorithm that checks for run time and general exceptions.
+
+    Args:
+        algorithm: object that extends TaskPerceiver
+
+    Example usage:
+        from sanity_test import sanity_test
+        import pytest
+        class Algorithm1(TaskPerceiver):
+            def analyze(self, frame, debug):
+                pass
+        class TestAlgorithm1():
+            @pytest.fixture(scope="class")
```

Comment:
I feel like `pytest.mark.parametrize` is better than using fixtures here.

Reply:
I was hoping for people to use the `algorithm` and `test_imgs` fixtures in other test functions too. Do you think it's better to specifically specify `@pytest.mark.parameterize("algorithm,test_imgs", [(Algorithm1(), get_test_imgs())]` for each test?

Reply:
That's how I did it during my internship. Then you don't need to deal with creating a test class and fixtures. You just need to provide some inputs and expected results to a test function.